builder/handlebars/kss-assets/kss.js

```diff
@@ -19,10 +19,16 @@
             for (idx = 0, _len2 = _ref2.length; idx < _len2; idx++) {
               rule = _ref2[idx];
               if ((rule.type === CSSRule.STYLE_RULE) && pseudos.test(rule.selectorText)) {
-                replaceRule = function(matched, stuff) {
-                  return matched.replace(/\:/g, '.pseudo-class-');
-                };
-                this.insertRule(rule.cssText.replace(pseudos, replaceRule));
+                selectorRules = rule.selectorText.split(',');
+                for(idx2 = 0, amountOfRules = selectorRules.length; idx2 < amountOfRules; idx2++) {
+                  var selector = selectorRules[idx2];
+                  if(pseudos.test(selector)) {
```

Comment:
If `selector === rule.selectorText` then the regexp test already has a state and will look for the next match in the string -- and fail.

Reply:
An alternative would be to drop the `g` flag of `pseudos`.